src/clarity-angular/popover/dropdown/dropdown-toggle.ts

```diff
@@ -30,7 +30,7 @@ export class DropdownToggle {
     }
 
     get active(): boolean {
-        return this._dropdown.open;
+        return this._dropdown.ifOpenService.open;
```

Comment:
I'd rather you inject the ifOpenService directly, even if we still need the dropdown itself for other features. Very small change, and probably subjective, but it "feels" cleaner.